src/gateway/WebSocketClient.ts

```diff
@@ -0,0 +1,139 @@
+import { EventEmitter } from 'events';
+import { Inflate } from 'zlib-sync';
+import ws from 'ws';
+import * as dgateway from 'discord-api-types/gateway/v8';
+import { Client } from '../clients/Client';
+import { WSEvents } from '../constants';
```

Comment:
```suggestion
import { WS_EVENTS } from '../constants';
```

Reply:
In this case : `WEBSOCKET_EVENTS` 😄

Reply:
(Also, this should be in the `gateway` folder, not in constants.ts